terraform/environment/dns_claim.tf

```diff
@@ -0,0 +1,62 @@
+data "aws_route53_zone" "claim-power-of-attorney-refund_service_gov_uk" {
+  provider = aws.old_refunds_production
+  name     = "claim-power-of-attorney-refund.service.gov.uk"
+}
+
+locals {
+  public_front_cloudfront_distribution_domain_name = "d3pm2jfxjwrqjc.cloudfront.net"
+  public_front_cloudfront_distribution_zone_id     = "Z2FDTNDATAQYW2"
```

Comment:
Could this not be a data source?

Reply:
No data source exists for cloudfront distrbutions